torch/testing/_comparison.py

```diff
@@ -824,6 +864,14 @@ def _compare_values(self, actual: torch.Tensor, expected: torch.Tensor) -> None:
         elif actual.layout == torch.jagged:
             actual, expected = actual.values(), expected.values()
             compare_fn = self._compare_regular_values_close
+        elif actual.dtype.is_floating_point and actual.dtype.itemsize == 1:
+            # intercept rtol and atol for bitwise comparison in low dimensional floats
+            def bitwise_comp(actual, expected, rtol=0.0, atol=0.0, equal_nan=False):
+                return self._compare_regular_values_close(
+                    actual, expected, rtol=0.0, atol=0.0, equal_nan=equal_nan
+                )
```

Comment:
Could we error out if the user passes a rtol or atol that is not 0.0 or the default value?

Also, based on the documentation, I would have expected that we already set the rtol/atol to 0.0 for float8 (https://pytorch.org/docs/stable/testing.html), what's going on there?